File: Service_eHospital_DongNai_A_NSTL/Implement/NS_NGOAINGU_UVService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_NGOAINGU_UVService : BaseService<NS_NGOAINGU_UV, INS_NGOAINGU_UVRepository>
    , INS_NGOAINGU_UVService
    {
    private readonly INS_NGOAINGU_UVRepository _NS_NGOAINGU_UVRepository;
    public NS_NGOAINGU_UVService(INS_NGOAINGU_UVRepository NS_NGOAINGU_UVRepository) : base(NS_NGOAINGU_UVRepository)
    {
    _NS_NGOAINGU_UVRepository = NS_NGOAINGU_UVRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_NGOAINGU_UV.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_NGOAINGU_UV } from './NS_NGOAINGU_UV.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_NGOAINGU_UVService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'id', 'MaUngVien', 'MaTDNgoaiNgu', 'Trinhdo', 'Nghe', 'Noi', 'Doc', 'Viet'];

    List: NS_NGOAINGU_UV[] | undefined;
    ListFilter: NS_NGOAINGU_UV[] | undefined;
    FormData!: NS_NGOAINGU_UV;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_NGOAINGU_UV";
    }
}

